### PR TITLE
feat: account profile - account link

### PR DESCRIPTION
**Pull request by stanleyyconsensys**
This PR is adding a Link component for account profile

when it clicked, it will redirect user to account profile page

it will display either user name or trimmed address when user name not given

### CODE REVIEW

File: src/utils/string.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * @returns The trimed text.
+ */
+export function trimText(content: string, length = 100) {
+  if (!content) {
+    return content;
+  }
+
+  const _content = content.trim();
+
+  const contentArr = [..._content];
```
Comment (ritave) [inline src/utils/string.ts:15]: Why is the conversion even needed
```typescript
function trimText(content: string, length = 100) {
  if (content.length < length) {
    return content;
  }
  return content.trim().slice(0, length) + '...';
}
```

File: src/components/AccountLink.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+export const AccountLink: FunctionComponent<AccountLinkProps> = ({
+  account,
+}) => {
+  let { name } = account;
+  name = name?.trim();
+
+  return (
+    <Link as={GatsbyLink} variant="landing" to={`/account/${account.address}`}>
+      {name && name.length > 0 ? name : trimAddress(account.address)}
```
Comment (ritave) [inline src/components/AccountLink.tsx:24]: Empty string is falsy.
```suggestion
      {name || trimAddress(account.address)}
```

File: src/utils/string.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+ * @param head - The heading length to render.
+ * @param tail - The tailing length to render.
+ * @returns The trimed text.
+ */
+export function trimTextByHeadTail(
+  content: string,
+  head: number,
+  tail: number,
+) {
+  if (!content) {
```
Comment (ritave) [inline src/utils/string.ts:37]: Same as above

**Comment by alainncls** [comment]
Closed in favour of #41

**Pull request closed without merging**